src/Illuminate/Notifications/NotificationSender.php

```diff
@@ -129,11 +130,12 @@ protected function preferredLocale($notifiable, $notification)
     /**
      * Send the given notification to the given notifiable via a channel.
      *
-     * @param  mixed  $notifiable
-     * @param  string  $id
-     * @param  mixed  $notification
-     * @param  string  $channel
+     * @param mixed $notifiable
+     * @param string $id
+     * @param mixed $notification
+     * @param string $channel
      * @return void
+     * @throws \Throwable
```

Comment:
Please undo all style changes to these docblocks

Reply:
done.